docs/readthedocs/source/doc/Orca/Tutorial/k8s.md

```diff
@@ -339,27 +327,35 @@ python /bigdl/nfsdata/train.py --cluster_mode k8s-cluster --data_dir /bigdl/nfsd
 
 ### 6.2 Use `spark-submit`
 
-Set the cluster_mode to "bigdl-submit" in `init_orca_context`.
-```python
-init_orca_context(cluster_mode="spark-submit")
-```
+If you prefer to use `spark-submit`, please follow the steps below to prepare the environment in the __Client Container__. 
 
-Pack the current activate conda environment to an archive in the __Client Container__:
-```bash
-conda pack -o environment.tar.gz
-```
+1. Set the cluster_mode to "spark-submit" in `init_orca_context`.
+    ```python
+    sc = init_orca_context(cluster_mode="spark-submit")
+    ```
+
+2. Download the requirement file from [here](https://github.com/intel-analytics/BigDL/tree/main/python/requirements/orca) and install the required Python libraries of BigDL Orca according to your needs.
+    ```bash
+    pip install -r /path/to/requirements.txt
+    ```
+    Note that you are recommended **NOT** to pip install BigDL Orca in the conda environment if you use spark-submit to avoid possible conflicts.
+
+3. Pack the current activate conda environment to an archive before submitting the example:
+    ```bash
+    conda pack -o environment.tar.gz
+    ```
 
 Some runtime configurations for Spark are as follows:
 
 * `--master`: a URL format that specifies the Spark master: k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port>.
 * `--name`: the name of the Spark application.
 * `--conf spark.kubernetes.container.image`: the name of the BigDL K8s Docker image.
-* `--conf spark.kubernetes.authenticate.driver.serviceAccountName`: the service account for the driver pod.
```

Comment:
why remove here?

Reply:
This is only needed for cluster mode, moved here and added in cluster mode arguments.